dist/yellow-text.js

```diff
@@ -252,6 +247,21 @@
 
       /**
       *
+      * hasContentChanged
```

Comment:
The name `hasContentChanged` is to continue the same nomenclature (verb+name+adjective), but is not problem if you prefer other name. :)